Packs/CortexResponseAndRemediation/Playbooks/playbook-CVE-2025-31324__-_SAP_NetWeaver_Visual_Composer.yml

```diff
@@ -0,0 +1,2554 @@
+      time_frame:
+        simple: 7 days ago
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3770.75,
+          "y": 2288
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "75":
+    id: "75"
+    taskid: a2837c0f-f130-495c-8c75-e9efdd51c1f7
+    type: condition
+    task:
+      id: a2837c0f-f130-495c-8c75-e9efdd51c1f7
+      version: -1
+      name: Is the integration of 'XQL Query Engine' available?
+      description: Returns 'yes' if integration brand 'XQL Query Engine' is available. Otherwise returns 'no'.
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "97"
+      "yes":
+      - "77"
+    scriptarguments:
+      brandname:
+        simple: XQL Query Engine
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 4191.5,
+          "y": 1720
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "77":
+    id: "77"
+    taskid: 995518bd-e0c4-407e-8ef0-d42de40f3c93
+    type: regular
+    task:
+      id: 995518bd-e0c4-407e-8ef0-d42de40f3c93
+      version: -1
+      name: Check XQL Quota
+      description: Retrieve the amount of query quota available and used.
+      script: '|||xdr-xql-get-quota'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "78"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3770.75,
+          "y": 1950
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "78":
+    id: "78"
+    taskid: 8079423c-9130-49c0-8fa3-65cf4c1fb18c
+    type: condition
+    task:
+      id: 8079423c-9130-49c0-8fa3-65cf4c1fb18c
+      version: -1
+      name: Is there enough quota?
+      description: "Is there enough quota?"
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "79"
+      "yes":
+      - "74"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: greaterThanOrEqual
+          left:
+            value:
+              complex:
+                root: PaloAltoNetworksXQL.Quota
+                accessor: license_quota
+                transformers:
+                - operator: subtraction
+                  args:
+                    by:
+                      value:
+                        simple: PaloAltoNetworksXQL.Quota.used_quota
```

Comment:
for some reason the transformer fails the task.
Maybe it returns a string and try to subtract an int?

Reply:
![image](https://github.com/user-attachments/assets/78242235-7d98-4abd-8417-dac36f5dcf9e)
How did you run it?
worked for me... can you show me your example?

Reply:
Found the issue, fixed that with extanded context 🦾